src/main/java/org/spongepowered/common/entity/player/SpongeUser.java

```diff
@@ -183,79 +184,83 @@ public DataContainer toContainer() {
                 .set(DataQueries.USER_SPAWNS, this.spawnLocations);
     }
 
+    private Player getPlayerIfOnline() {
```

Comment:
If the exception message is going to mention the inventory, this method should be named differently.